indigo_app/templates/place/detail.html

```diff
@@ -148,8 +161,24 @@ <h5 class="mb-0">Current tasks ({{ total_open_tasks }})</h5>
           <h5 class="mb-0">Recent activity</h5>
         </div>
         <div class="card-body">
-          <div class="p-relative" style="height: 100px">
-            <canvas id="activity-chart" data-values="{{ activity_history }}"></canvas>
+          <div>
+            <div class="p-relative" style="height: 100px">
+              <canvas id="activity-chart" data-values="{{ activity_history }}"></canvas>
+            </div>
+          </div>
+          <div class="d-flex flex-row justify-content-between">
+            <div class="px-4 mt-3 text-center bg-light border rounded">
```

Comment:
Move the `mt-3` onto the `d-flex` wrapping div, then you don't have to replicate it.